pkg/cmd/cli/cmd/newapp.go

```diff
@@ -845,3 +849,14 @@ func (r *configSecretRetriever) CACert() (string, error) {
 	}
 	return "", nil
 }
+
+func checkGitInstalled(w io.Writer) {
+	gitBinary := "git"
+	if goruntime.GOOS == "windows" {
+		gitBinary = "git.exe"
+	}
+	_, err := exec.LookPath(gitBinary)
+	if err != nil {
+		fmt.Fprintf(w, "warning: Cannot find git. Ensure that it is installed and in your path.\n")
```

Comment:
As for the message itself, does this work? 
"warning: Cannot find git. Ensure that it is installed and in your path. Git is required to work with git repositories.\n"